### ISSUE TITLE
Markdown Input behaves weirdly when content contains double spaces

### ISSUE THREAD

**Issue by chris-ware** [filamentphp/filament#4295]
### Package

filament/filament

### Package Version

v2.16.19

### Laravel Version

v9.32.0

### Livewire Version

v2.10.7

### PHP Version

PHP 8.1.8

### Problem description

When a markdown input contains a double space, the markdown input behaves very weirdly:
![Screenshot 2022-09-30 at 11 14 19](https://user-images.githubusercontent.com/19684457/193249239-b6822e3a-53d3-4fee-bb32-3d783b0bc9ce.png)

### Expected behavior

Markdown field to behave as expected
![Screenshot 2022-09-30 at 11 19 30](https://user-images.githubusercontent.com/19684457/193249361-a2078222-79e8-4b57-ac47-28e81890ef99.png)

### Steps to reproduce

Checkout reproduction repo, add user, then seed. View created user in edit mode.

### Reproduction repository

https://github.com/chris-ware/filament-markdown-issue.git

**Comment by Kristories** [comment]
I tested from your reproduction repository. This issue is because you are using `\r\n`, whereas if you try to save via form, you will get `\n`. Why not change your seeder?

Do we have to replace `\r\n` to `\n` via component @danharrin?

![](https://images.squarespace-cdn.com/content/v1/518f5d62e4b075248d6a3f90/1391582191233-V1NR6PUP72BQLICE6BTG/image-asset.jpeg?format=1500w)

**Comment by chris-ware** [comment]
The purpose of the seeder is to show the issue, so replacing it does nothing.

**Comment by Kristories** [comment]
`LF` and `CR` are [control characters](https://en.wikipedia.org/wiki/Control_character) or [bytecode](https://en.wikipedia.org/wiki/Bytecode) that can be used to mark a line break in a text file.

- `LF` = Line Feed (`\n`, `0x0A` in hexadecimal, 10 in decimal) — **moves the cursor down to the next line** without returning to the beginning of the line.
- `CR` = Carriage Return (`\r`, `0x0D` in hexadecimal, 13 in decimal) — **moves the cursor to the beginning of the line** without advancing to the next line.


Did you see the context? I tried with demo.

#### LF 
```
process.stdout.write('long message first\n')
process.stdout.write('short message\n')
```

```
long message first
short message
```

#### CF 
```
process.stdout.write('long message first\r')
process.stdout.write('short message\n')
```

```
short messagefirst
```

**Comment by danharrin** [comment]
Fixed by #4360
